src/main/scala/com/lunatech/pointingpoker/RoomManager.scala

```diff
@@ -49,11 +49,11 @@ class RoomManager extends Actor with ActorLogging {
       }
   }
 
-  private def createRoom(roomId: UUID): ActorRef = {
+  def createRoom(roomId: UUID): ActorRef = {
     context.actorOf(Room.props(roomId), roomId.toString)
   }
 
-  private def handleIncomeMessage(room: ActorRef, message: WSMessage): Unit =
+  def handleIncomeMessage(room: ActorRef, message: WSMessage): Unit =
     message.messageType match {
       case MessageType.Init      => // Should never arrive here
```

Comment:
These 3 messages aren't expected here but they might arrive anyway if someone makes a mistake. I would suggest logging some errors instead; it would make debugging much easier.